pages/wiki/ip-connection.md

```diff
@@ -40,14 +40,39 @@ Some more <a href="https://wiki.archlinux.org/index.php/ConnMan#Connecting_to_a_
 
 *****
 
+It is possible to allow your watch to be able to use your Linux computer's internet connection via a USB connection.  You will need:
 
-Configure a NAT **on your computer** (Note: Replace eth0 with the name of the interface that connects your computer to the Internet) with:
+ 1. a Linux computer with RNDIS support loaded and enabled
+ 2. the ability to change network configurations on that computer
+ 3. a network connection
```

Comment:
I'd start the list items with capital letters. I can see that they all perfectly add to the "You will need:" sentence. But imo those list items are still individual new sentences just judging from the numeration breaking the text flow.

Reply:
Done.